organize/forms.py

```diff
@@ -113,7 +117,16 @@ def clean_date(self):
         date = self.cleaned_data.get('date')
         validate_approximatedate(date)
         # TODO: add checking if the event is in the future
```

Comment:
Looks easy enough to implement in `validate_event_date` while you're at it. In fact, I think it should already be solved since this evaluates to `True`.
```python
date(2020, 10, 20) - date.today() < timedelta(days=90)                                                                                                                       
```